### PR TITLE
Simplified README to minimize duplication with official docs

### PR DESCRIPTION
**Pull request by dylan-smith**
Issue #327 

Tried to keep the documentation in our readme to a bare minimum to minimize duplication with our official docs.

This will hopefully make it easier for us to keep our docs up to date as we roll out new features, as there will only be one place to change.

- [x] Did you write/update appropriate tests
- [x] Release notes updated (if appropriate)
- [x] Appropriate logging output
- [x] Issue linked
- [x] Docs updated (or issue created)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,10 @@
...
+The [GitHub Enterprise Importer](https://docs.github.com/en/early-access/github/migrating-with-github-enterprise-importer) (GEI, formerly Octoshift) is a highly customizable API-first migration offering designed to help you move your enterprise to GitHub Enterprise Cloud. The GEI-CLI wraps the GEI APIs as a cross-platform console application to simplify customizing your migration experience.
 
 > GEI is in a private preview for GitHub Enterprise Cloud. It needs to be enabled before using this CLI. Please reach out to [GitHub Sales](https://github.com/enterprise/contact) to enquire about getting into the private beta. 
 
-This version of the GEI-CLI is informally maintained by GitHub. However, this is **not a supported GitHub product**. Customers leveraging these tools must understand that any support must come through a paid GitHub Expert Services engagement.
-
 ## Supported Scenarios
```
Comment (roferg) [inline README.md:10]: We should just replace supported scenarios all up with a link to the docs.

File: README.md [hunk 2]
```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
...
-                        Note: Expects GH_PAT env variable to be set.
-```
-
-To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
-
-## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+## Using the GEI CLI
+There are 2 separate CLI's that we ship:
+- `ado2gh` - Intended for migrations from Azure DevOps -> GitHub
+- `gh gei` - Intended for GitHub -> GitHub migrations
```
Comment (roferg) [inline README.md:22]:
```suggestion
- `gh gei` - Intended for running migrations from anywhere supported to GitHub. 
```

File: README.md [hunk 3]
```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
...
-  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
-                        Note: Expects GH_PAT env variable to be set.
-```
-
-To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
-
-## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+## Using the GEI CLI
+There are 2 separate CLI's that we ship:
+- `ado2gh` - Intended for migrations from Azure DevOps -> GitHub
```
Comment (roferg) [inline README.md:21]:
```suggestion
- `ado2gh` - Extends the basic migration support in the `gh gei` CLI to include extra commands like re-wiring Azure Pipelines and Boards after a migration is complete. 
```

File: README.md [hunk 4]
```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
...
-  migrate-repo          Invokes the GitHub APIs to migrate the repo and all repo data.
-  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
-                        Note: Expects GH_PAT env variable to be set.
-```
-
-To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
-
-## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+## Using the GEI CLI
+There are 2 separate CLI's that we ship:
```
Comment (roferg) [inline README.md:20]: I'd reverse the ordering of the bullet points below.

File: README.md [hunk 5]
```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
...
 
-General usage will use the `generate-script` command to create a script that can be used to migrate all repositories from a GHES organization.
+To use `ado2gh` download the latest version from the [Releases](https://github.com/github/gh-gei/releases/latest) in this repo.
 
-Due to many GHES instances containing firewall rules restricting the direct API access that GEI needs to perform a migration, the GEI CLI uploads the migration's archive data to Azure Blob Storage as a step prior to starting the migration using GEI. This allows GEI to perform a migration without directly accessing the GHES instance. To get started you'll need to setup an [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal). Then, you'll pass in a [connection string](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/api/connection-strings/storage-connection-strings) to the CLI for that Azure storage account, specified in the parameters below. GEI CLI will take care of the rest.
+To use `gh gei` first install the latest [GitHub CLI](https://github.com/cli/cli#installation), then run the command
```
Comment (roferg) [inline README.md:26]: Any references to how to use the `gh gei` cli should be removed and just tell folks to read the docs. We should only list ado2gh extra commands here.

File: README.md [hunk 6]
```diff
@@ -18,164 +16,67 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
...
 
-Once you have configured the `octoshift`(*gh-gei*) command-line tool and `environment variables` for the person access tokens, you can run the command-line tool and see all available options.
-![octoshift help](https://user-images.githubusercontent.com/29484535/145065029-ea8b3fcd-fcea-4f9b-ba7e-f9d3407e17fa.png)
+### Migrating GitHub to GitHub
+The quick start video below will help you start migrating between GitHub organizations. 
 
-The first step you will want to run is `generate-script` to help outline all commands to migrate an entire **Azure Project**
-This command will generate a `migrate.ps1` file in the local folder. You will want to open it with an editor tool as it can be quite large. It's recommended that you include the `--repos-only` flag the first time. 
-Tools like `Atom`, `VSCode`, or `NotePad++` are great ways to see the data.
+* Migrating GitHub to GitHub with the GEI CLI: https://youtu.be/5cQkM_8n5YY
```
Comment (roferg) [inline README.md:74]: We need to remove this video as it mentions using `.sh` extensions.

**Review by roferg — CHANGES_REQUESTED** [review]
We direct folks to the docs instead of explaining how to use the `gh gei` cli here.

File: README.md (hunk 1 above)
Comment (dylan-smith) [reply README.md:10]: sounds good, I'll just remove this section. There are plenty of links in here to the official docs already

File: README.md (hunk 6 above)
Comment (dylan-smith) [reply README.md]: done, I plan to re-record all videos hopefully next week

**Review by roferg — APPROVED** [review]
Let's remove the gh -> gh video for now until we can re-record it.

File: README.md (hunk 5 above)
Comment (dylan-smith) [reply README.md:17]: I'd rather not delete the small sections on usage, I scaled them way back from like 10 pages to 1 page, but I think we should have some extremely minimal "here's how you use this, see the docs for more" type section in the README. I feel that's expected in a typical OSS readme.

I was just trying to basically say "install it from here, run generate-script, then run the script. See the docs for more"

**Pull request merged**